docs/useful_tools.md

```diff
@@ -0,0 +1,140 @@
+Apart from training/testing scripts, We provide lots of useful tools under the `tools/` directory.
+
```

Comment:
toc for this file?

Reply:
It will be ugly in readthedocs in this page if toc is added to this file.

Reply:
screenshot?

Reply:
![ugly_readthedocs](https://user-images.githubusercontent.com/33949370/99221627-e0762800-281b-11eb-92f3-c081ba03a2ed.png)

Reply:
How does the `# Tutorial 1: Finetuning Models` page looks like? It does not have second level items also

Reply:
![beaultiful_tutorial](https://user-images.githubusercontent.com/33949370/99224010-7c099780-2820-11eb-858a-bb588b0deb75.png)

Reply:
They should be similar, right?